### DISCUSSION TITLE
Welcome to napi-rs Discussions!

### DISCUSSION THREAD

**Discussion by Brooooooklyn** [napi-rs/napi-rs#348] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tyrone-sudeium** [comment]
I'll break the ice, since I just wanted to say thank you SO much for this project! I've been a Node.js user since 2012 and I've always been terrified to try writing native extensions for it because the build tooling and associated ecosystem around it (for C and C++) is nightmarish, despite being quite comfortable with C and C++ normally.

Thanks to this project I've finally written my own extension, AND because your template was so good it automatically got good CI/CD! Thanks to rust-analyzer the development experience on the rust side was also top tier. You are an absolute legend. I hope when my rust abilities improve I'll be able to make my own contributions to this (and maybe skia) in the future!

**Reply by Brooooooklyn** [reply]
Thank you for your praise, I am honored that this project can help you 🚀
